test/test_tensor.py

```diff
@@ -134,43 +148,50 @@ def test_slice_1(stop):
 @pytest.mark.parametrize("start", [0, 1, 2, 10])
 @pytest.mark.parametrize("stop", [0, 1, 2, 10])
 @pytest.mark.parametrize("step", [1, 2, 5, 10])
-def test_slice_2(start, stop, step):
-    t = torch.randn(10, 2)
+@pytest.mark.parametrize("backend", ["torch", "numpy"])
+def test_slice_2(start, stop, step, backend):
+    t = randn((10, 2), backend)
     actual = Tensor(t)["i"](i=Slice("j", start, stop, step, dtype=10))
     expected = Tensor(t[start: stop: step])["j"]
     assert_close(actual, expected)
 
 
-def test_arange_simple():
-    t = torch.randn(3, 4, 5)
+@pytest.mark.parametrize("backend", ["torch", "numpy"])
+# FIXME: this test is failing at the second `assert_close`?
```

Comment:
I am not sure why this simple test is failing while most of remaining tests passed. :(